src/config/audiences.js

```diff
@@ -1,4 +1,14 @@
 export default {
+  bus: {
+    production: {
+      clientId: 'bmoJ2jhfyIP5s4IZx5Ss6xQ9lmkoDixO',
+      host: 'https://bus.ndustrial.io'
+    },
+    staging: {
+      clientId: '8qY2xJob1JAxhmVhIDLCNnGriTM9bct8',
```

Comment:
Oh shoot, this ID I gave you is the ID used by Contxt coordinator's staging instance. The client ID of the staging instance is `1HD1NG1VTBtkqRt2HRRj3E3hdmqmwzoz`

Reply:
Got it. Thx! :D